rpcwebsocket.go

```diff
@@ -1737,15 +1736,14 @@ out:
 		case <-ntfnSentChan:
 			// No longer waiting if there are no more messages in
 			// the pending messages queue.
-			next := pendingNtfns.Front()
-			if next == nil {
+			if len(pendingNtfns) == 0 {
 				waiting = false
 				continue
 			}
-
 			// Notify the outHandler about the next item to
 			// asynchronously send.
-			msg := pendingNtfns.Remove(next).([]byte)
+			msg := pendingNtfns[0]
+			pendingNtfns = pendingNtfns[1:]
```

Comment:
set pendingNtfns[0] to nil before this and after reading msg so it's no longer referenced by the backing slice and may be collected later.